src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1752,6 +1752,8 @@ open_common(int (*real_open)(const char *pathname, int oflags, ...), const char
 
 	if (!is_target_path)
 		goto org_func;
+	if (oflags & O_APPEND)
```

Comment:
I was thinking you could add that in dfs (in open_file)

Reply:
also a test for this to make sure it returns ENOTSUP :-)

Reply:
ok. I will move the change to dfs.c as well as adding a test. Thank you!